src/main/java/com/openlattice/data/DataApi.java

```diff
@@ -160,10 +155,10 @@ Integer replaceEntityProperties(
      * @param associations Set of associations to create. Keys are association entity set ids and values for each keys
      *                     are the data to be created.
      */
-    @POST( BASE + "/" + ASSOCIATION )
-    ListMultimap<UUID, UUID> createAssociations( @Body ListMultimap<UUID, DataEdge> associations );
+    @POST( BASE + ASSOCIATION )
+    Map<UUID, List<UUID>> createAssociations( @Body Map<UUID, List<DataEdge>> associations );
```

Comment:
@katherinebernstein Can you pls check for the same thing as in `createEntityAndAssociationData`, whether we have values like
`{a:[1], a:[2], a:[3]}` or `{a:[1,2,3]}` ?

Reply:
Btw, is it even possible to have an object in javascript with same keys but different values (as in possible JSON representation of multimap)?

Reply:
As far as I found out, when we validate GraphData on frontend side, we allow empty array values, since it only checks if it's an array or not and lodash.isArray([]) is true